docs/tdvt.md

```diff
@@ -272,6 +283,9 @@ Run TDVT from your working directory since it will need the TDS and INI files yo
 
 To show the registered datasources and suites, run: `tdvt --list`
 
+To run smoke tests for a data source:
```

Comment:
Maybe a quick explanation of what it is and why you would want to use it? Something like:
"Run this to ensure TDVT can connect to your database properly."